backend/server/src/handler/rating.rs

```diff
@@ -1,20 +1,20 @@
 use crate::models::app::AppState;
 use crate::models::auth::{
     ApplicationCreatorGivenApplicationId, ApplicationReviewerGivenApplicationId,
-    ApplicationReviewerGivenRatingId, RatingCreator, SuperUser,
+    ApplicationReviewerGivenRatingId, RatingCreator,
 };
 use crate::models::error::ChaosError;
-use crate::models::ratings::{NewRating, Rating};
+use crate::models::rating::{NewRating, Rating};
 use crate::models::transaction::DBTransaction;
 use axum::extract::{Json, Path, State};
 use axum::http::StatusCode;
 use axum::response::IntoResponse;
 
-pub struct RatingsHandler;
+pub struct RatingHandler;
 
-impl RatingsHandler {
+impl RatingHandler {
     // TODO: are all the user permissions as required? Who should be able to do what with ratings?
-    pub async fn create_rating(
+    pub async fn create(
         State(state): State<AppState>,
         Path(application_id): Path<i64>,
         _admin: ApplicationCreatorGivenApplicationId,
```

Comment:
```suggestion
        _admin: ApplicationCreatorGivenApplicationId,
```

Didn't catch this earlier but this naming should be RatingCreatorGivenApplicationId